torchtitan/experiments/kernels/moe/indices.py

```diff
@@ -194,18 +218,22 @@ def generate_permute_indices(
             tokens_per_expert_group,
             start_index_values,
             write_offsets,
+            total_tokens_per_expert,  # Pass to check for zero tokens
             experts_per_rank,
             num_ranks,
             max_len,
+            min_slots_per_expert,
         )
     else:
         permuted_indices = fill_indices_wrapper(
             tokens_per_expert_group,
             start_index_values,
             write_offsets,
+            total_tokens_per_expert,  # Pass to check for zero tokens
             experts_per_rank,
             num_ranks,
             max_len,
+            min_slots_per_expert,
```

Comment:
`fill_indices_wrapper` doesn't have `min_slots_per_expert` argument https://github.com/pytorch/torchtitan/pull/1166/files?diff=split&w=1#diff-586d464a893e9ca89d1ec981485e85e01bf1f6d600bbff3554f578e2e0201ec0R85, you are overwriting default value of `block_size` here, is it what you want? If it is, it's very confusing

Reply:
thanks - this was a mistake when I was trying to clean up, and already fixed earlier. 
* we don't need the min_slots_per_expert arg for the kernel so I removed but in an incomplete fashion.

Reply:
oops...I didn't fully check this fix in earlier (staged but not commited)...anyway, thanks for the catch, updated and resolved.